packages/dropdown-react/package.json

```diff
@@ -17,7 +17,7 @@
     },
     "dependencies": {
         "@fremtind/jkl-dropdown": "^0.0.1",
-        "downshift": "^3.2.10"
+        "@nrk/core-toggle": "^3.0.4"
```

Comment:
Shouldn't we contain the dropdown changes in the dropdown PR?

Reply:
Yes. I'm not sure how this snuck in 😑